channels.yaml

```diff
@@ -0,0 +1,11 @@
+# Example channels config
```

Comment:
Is this an example or an actual config?

Reply:
i simultaneously noticed that as well. removed it. copied from the k3s config (which should also drop this comment)